app/views/pages/formatting.html.erb

```diff
@@ -49,6 +49,13 @@
 2. <%= t '.lists_code3' %>
 3. <%= t '.lists_code4' %></pre>
 
+      <h3><%= t '.quote' %></h3>
+      <p><%= t '.quote_ex1' %></p>
+      <pre><%= t '.quote_code1' %>
+&gt; <code><%= t '.quote_code2' %></code>
```

Comment:
I was expecting the `>`  character to be interpreted by HTML so I used its raw code : `&gt;`. I must admit I've not been able to test whether the rendering is correct though.